lib/ansible/modules/windows/win_domain_group_membership.ps1

```diff
@@ -39,6 +39,8 @@ if ($null -ne $domain_server) {
     $extra_args.Server = $domain_server
 }
 
+$name = Get-ADGroup $name @extra_args
```

Comment:
Probably better not to override the `$name` argument as that's an input from the module options. I would recommend you do `$group` or something to differentiate between them. Also would be nice if you added the parameter name i.e. `-Identity $name`

Reply:
Changed $name to $ADGroup so it is better to distinguish between the Group Name and the Object